WhisperAPI/WhisperAPI/Services/Suggestions/SuggestionsService.cs

```diff
@@ -104,6 +113,15 @@ public bool IsQueryRelevant(SearchQuery searchQuery)
             return coveoIndexDocuments.Where(x => !queries.Any(y => y.Contains(x.Uri)));
         }
 
+        private static IEnumerable<Question> FilterOutChosenQuestions(
+            ConversationContext conversationContext,
+            IEnumerable<Question> questions)
+        {
+            var questionsText = conversationContext.SelectedQuestions.Select(x => x.Text);
+
+            return questions.Where(x => !questionsText.Any(y => y.Contains(x.Text)));
```

Comment:
Contains ? why not equal ?
Even if I doubt that it could really happen, if a question is a substring of another it will be filter out, is this by desing ?